roles/satellite-hostname/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: "check that variables are updated"
-  fail: msg="Please update the variables in roles/change_hostname/vars/main.yml"
+  fail: msg="Please update the variables in satellite-hostname-vars.yml"
```

Comment:
I may be wrong, but consider https://github.com/RedHatSatellite/satellite-clone/pull/106/files#r99473045 here as well.